include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1473,7 +1473,7 @@ class MatrixFree : public Subscriptor
     const unsigned int                           dof_handler_index = 0) const;
 
   /**
-   * In the hp adaptive case, return number of active_fe_indices.
+   * In the hp-adaptive case, return number of active_fe_indices.
```

Comment:
```suggestion
   * In the hp-adaptive case, return number of active FE indices.
```

Sorry again for being pedantic

In #11216, we also decided to remove underscores from `active_fe_indices` and write it as `active FE indices`. While there, would you mind changing this as well?

Reply:
I just noticed that the following lines have the same problem. These functions have been introduced in #11260, so after I ran my scripts :(

Reply:
Let's do that in a follow up, if you don't mind. I've got other stuff going on in my tree right now.